pkg/hub/submarinerbrokerinfo/brokerinfo.go

```diff
@@ -386,13 +405,13 @@ func getTokenAndCAFromSecret(ctx context.Context, tokenSecret *corev1.Secret, ku
 	return string(tokenSecret.Data["token"]), base64.StdEncoding.EncodeToString(kubeAPIServerCA), nil
 }
 
-func GenerateBrokerName(clusterSetName string) string {
-	name := fmt.Sprintf("%s-%s", clusterSetName, brokerSuffix)
-	if len(name) > namespaceMaxLength {
-		truncatedClusterSetName := clusterSetName[(len(brokerSuffix) - 1):]
+func GenerateBrokerName(name string) string {
```

Comment:
Why is this changing?

Reply:
The passed argument was named `clusterSetName` Now we're reusing it to also generate similar name with `clusterName` as an argument. So just changed the argument name to make it more generic and remove assumption that it will always be a clusterSetName.